backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -422,32 +423,33 @@ describe('ContributionResolver', () => {
         resetToken()
       })
 
-      describe('wrong contribution id', () => {
```

Comment:
Don't understand why you need to change the order makes it harder to read the changes.
I first thought you removed it!

Reply:
Because thats how the code is
![image](https://user-images.githubusercontent.com/1238238/217364242-1a9c0b9d-f4b5-475d-a96d-dfc83ad1365d.png)

Reply:
OK!